src/tools/debugger/SteppingStrategy.java

```diff
@@ -0,0 +1,44 @@
+package tools.debugger;
+
+import som.vm.Activity;
+
+public abstract class SteppingStrategy {
```

Comment:
Hm, we should look into that in more detail. I don't think you need a source section, tags, and nodes.

I don't need them for my breakpoints, and I don't see why yours would be so fundamentally different that you need all that. But perhaps I am missing something.